.github/workflows/release.yaml

```diff
@@ -37,4 +37,4 @@ jobs:
           git add -A "charts/**/README.md"
           git add -A "charts/**/values.yaml"
           git pull && git commit -m "Updating README.md and values files" && git push
-          popd
+          popd
```

Comment:
just a nit but is it needed to remove the new line at the EOF.

Reply:
no particular reasons. I tried adding things below and then removed them later thus why.